### ISSUE TITLE
Loss is down but high error on next(iter(dataloader)) persists

### ISSUE THREAD

**Issue by rubnCarmona2** [rubnCarmona2/manifold-worms#1]
Not sure if this is an issue of training or preprocessing. It could as well be due to the current implementation only allowing batch_size=1.

**Comment by rubnCarmona2** [comment]
MW loss seems to follow baseline neural network's loss on the same task with same batch size.
